estate_account/models/estate_property.py

```diff
@@ -0,0 +1,29 @@
+from odoo import models, Command
+from odoo.exceptions import UserError
+
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def action_sold(self):
+        if not (self.buyer and self.selling_price):
+            raise UserError('Customer or selling price was not set.')
```

Comment:
I noticed your error messages are not getting translation

Add `_()` -> `from odoo import _` to make them translatable